pkg/reconciler/ingress/config/istio.go

```diff
@@ -39,6 +39,10 @@ const (
 
 	// MeshGatewayName is the name of the special 'mesh' Istio Gateway.
 	MeshGatewayName = "mesh"
+
+	// ReconcileExternalGatewayKey the is the name of the configuration entry that specifies
+	// reconciling external Istio Gateways or not.
+	ReconcileExternalGatewayKey = "reconcileExternalGateway"
```

Comment:
A dumb question but why is this 'external'? I thought this reconciliation will only happen for the `knative-ingress-gateway`